<ui version="4.0" >
 <class>FileTypeConfigWidget</class>
 <widget class="QWidget" name="FileTypeConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>461</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblFiletype" >
       <property name="text" >
        <string>&amp;Filetype:</string>
       </property>
       <property name="buddy" >
        <cstring>cmbFiletypes</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="cmbFiletypes" >
       <property name="whatsThis" >
        <string>Select the filetype you want to change.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNew" >
       <property name="whatsThis" >
        <string>Create a new file type.</string>
       </property>
       <property name="text" >
        <string>&amp;New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete" >
       <property name="whatsThis" >
        <string>Delete the current file type.</string>
       </property>
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbProperties" >
     <property name="title" >
      <string>Properties</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lblName" >
        <property name="text" >
         <string>&amp;Name:</string>
        </property>
        <property name="buddy" >
         <cstring>edtName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KLineEdit" name="edtName" >
        <property name="whatsThis" >
         <string>The name of the filetype will be the text of the corresponding menu item.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblSection" >
        <property name="text" >
         <string>&amp;Section:</string>
        </property>
        <property name="buddy" >
         <cstring>edtSection</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KLineEdit" name="edtSection" >
        <property name="whatsThis" >
         <string>The section name is used to organize the file types in menus.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblVariables" >
        <property name="text" >
         <string>&amp;Variables:</string>
        </property>
        <property name="buddy" >
         <cstring>edtVariables</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KLineEdit" name="edtVariables" >
        <property name="whatsThis" >
         <string>&lt;p>This string allows to configure Kate's settings for the files selected by this mimetype using Kate variables. Almost any configuration option can be set, such as highlight, indent-mode, encoding, etc.&lt;/p>&lt;p>For a full list of known variables, see the manual.&lt;/p></string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="lblHl" >
        <property name="text" >
         <string>&amp;Highlighting:</string>
        </property>
        <property name="buddy" >
         <cstring>edtFileExtensions</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="lblExtensions" >
        <property name="text" >
         <string>File e&amp;xtensions:</string>
        </property>
        <property name="buddy" >
         <cstring>edtFileExtensions</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="KLineEdit" name="edtFileExtensions" >
        <property name="whatsThis" >
         <string>The wildcards mask allows to select files by filename. A typical mask uses an asterisk and the file extension, for example &lt;code>*.txt; *.text&lt;/code>. The string is a semicolon-separated list of masks.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="lblMimeTypes" >
        <property name="text" >
         <string>MIME &amp;types:</string>
        </property>
        <property name="buddy" >
         <cstring>edtMimeTypes</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="KLineEdit" name="edtMimeTypes" >
          <property name="whatsThis" >
           <string>The mime type mask allows to select files by mimetype. The string is a semicolon-separated list of mimetypes, for example &lt;code>text/plain; text/english&lt;/code>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnMimeTypes" >
          <property name="whatsThis" >
           <string>Displays a wizard that helps you easily select mimetypes.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="lblPriority" >
        <property name="text" >
         <string>P&amp;riority:</string>
        </property>
        <property name="buddy" >
         <cstring>sbPriority</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QSpinBox" name="sbPriority" >
        <property name="whatsThis" >
         <string>Sets priority for this file type. If more than one file type selects the same file, the one with the highest priority will be used.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="KComboBox" name="cmbHl" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>351</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDownload" >
       <property name="text" >
        <string>Download Highlighting Files...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>160</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
